### PR TITLE
Add `o!ET 2021` tournament article

### PR DESCRIPTION
**Pull request by fajar13k**
- Add osu! European Tournament 2021 tournament article
- Tournament already ended

### CODE REVIEW

File: wiki/Tournaments/o!ET/2021/en.md [hunk 1]
```diff
@@ -0,0 +1,511 @@
...
+### General rules
+
+1. The tournament uses a 4v4 format, with drafted teams of 8. Each team will be composed of 2 players per tier.
+   - Each tier has its own rank range. They go as follows:
+     - Tier 1: #1 - #2,500
+     - Tier 2: #2,501 - #7,500
+     - Tier 3: #7,501 - #15,000
+     - Tier 4: #15,001 - #30,000
+     - There will not be a rank buffer for any of the ranges
+2. Only players from the Europe region may sign up to the tournament. Participants may find a list of European countries through [following link](https://en.wikipedia.org/wiki/List_of_European_countries_by_area). If they wish to enter the tournament while the flag of their profile page has isn't listed here, participants can contact the host through Discord DMs to discuss the matter.
```
Comment (TPGPL) [inline wiki/Tournaments/o!ET/2021/en.md:429]:
```suggestion
2. Only players from the Europe region may sign up to the tournament. Participants may find a list of European countries through [following link](https://en.wikipedia.org/wiki/List_of_European_countries_by_area "Wikipedia"). If they wish to enter the tournament while the flag of their profile page has isn't listed here, participants can contact the host through Discord DMs to discuss the matter.
```

File: wiki/Tournaments/o!ET/2021/en.md [hunk 2]
```diff
@@ -0,0 +1,511 @@
...
+3. If everything goes as planned, the entire draft will be held using the bot, and the following steps will apply. In the case the bot is not ready, everything will be done manually, but not much will change.
+4. The draft will begin with shdewz-chan mentioning the captain who is to pick. Captains have **2 minutes** to decide on which player to pick, using a command provided to them. Once a captain has picked, shdewz-chan will mention the next captain to pick. This will go on until there are no players left to pick.
+5. Captains may freely pick from any tier, as long as their team does not have more than 2 players from the same tier.
+6. Pick order will be 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16. Once the 16th captain on the list has been reached, pick order will reverse until we reach the first name on the original list again (16, 15, 14, and so on... in a snake order draft). 1st and 16th names will get to pick twice (this rule won't apply for the first round of picks for 1st name, and for the last round of picks for the 16th name (as there will only be one player left to pick anyway).
+
+### Match procedures & Mappols format
```
Comment (TPGPL) [inline wiki/Tournaments/o!ET/2021/en.md:453]:
```suggestion
### Match procedures & Mappools format
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/o!ET/2021/en.md [hunk 3]
```diff
@@ -0,0 +1,511 @@
...
+   - During a Tiebreaker pick, players can, but are not obliged to, select mods to play the map.
+   - In the case of a tie in score, the Tiebreaker will be played (e.g., in a best of 11 match and 5-5 scenario, the tiebreaker will be played).
+5. Team captains will roll to determine the pick and ban order. The team that rolls the highest will ban in second and pick first.
+6. Teams will be allowed to ban 1 map per match up to Quarterfinals and Loser's Round 1, and 2 maps from Semifinals and onward. Ban alternation will be allowed, if the pool supports 2 bans. However, they cannot ban two maps from the same mod pool, with the exception of Freemod.
+7. The winning condition, mappool format and bans amount for each stage are listed as follows:
+   - The Round of 16 and the Quarterfinals: Best-Of-9 (First to 5 points wins), 1 ban per team. Pool format: 5FM/3FO/3DT/3TB.
+   - The Semifinals: Best-Of-11 (First to 6 points wins), 2 bans per team. Pool format: 7FM/4FO/4DT/3TB.
+   - The Finals and the Grand Finals: Best-Of-13 (First to 7 points wins), 2 bans per team. Pool format: 8FM/5FO/5DT/3TB.
```
Comment (Walavouchey) [inline wiki/Tournaments/o!ET/2021/en.md:484]:
```suggestion
   - The Round of 16 and the Quarterfinals: Best of 9 (first to 5 points wins), 1 ban per team. Pool format: 5FM/3FO/3DT/3TB.
   - The Semifinals: Best of 11 (first to 6 points wins), 2 bans per team. Pool format: 7FM/4FO/4DT/3TB.
   - The Finals and the Grand Finals: Best of 13 (first to 7 points wins), 2 bans per team. Pool format: 8FM/5FO/5DT/3TB.
```

File: wiki/Tournaments/o!ET/2021/en.md [hunk 4]
```diff
@@ -0,0 +1,511 @@
...
+8. Regarding disconnections/aborts: they can only be made up to 30 seconds after a map’s start. If the map is aborted/a disconnection happens:
+   - In the first 30 seconds: Map will be aborted, and the played map will be restarted.
+   - After the first 30 seconds: Map **will not** be aborted, and it is up to the team that wasn’t affected by the abort to decide whether they wish to replay the map or not.
+   - Each team can call for an abort one time per match.
```
Comment (Walavouchey) [inline wiki/Tournaments/o!ET/2021/en.md:488]:
```suggestion
8. Disconnections and aborts can only be made up to 30 seconds after a map’s start for a replay of that map. After 30 seconds, the map **will not** be aborted, and the team that wasn’t affected by the abort decides whether they wish to replay the map or not. Each team can call for an abort one time per match.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]
Bit of a writing pass. Tournament rules do seem to get very long-winded...

Additionally, please replace all occurrences of `’` (right single quotation mark) with `'` (apostrophe)

**Review by TPGPL — CHANGES_REQUESTED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**